### PR TITLE
Display absolute paths when displaying the output of ls and find.

### LINKED ISSUE #839: There are more commands where the path lacks the initial / err filepath.Separator
Both find and ls seem to be missing the same / prefix.

## Output of `restic version`
restic 0.4.0 (v0.4.0-79-g1d64a1d-dirty)
compiled with go1.8 on linux/amd64

## Expected behavior
```shell
$ ./restic -r rest:http://qnap.********:8000 find -s 0eb7a75d bin
found 55 matching entries in snapshot 0eb7a75d607a6dffd4f9342e2524326b23151d57875131a368ceb9f17d2b51c3
  drwxr-xr-x     0     0      0 2017-01-29 13:21:28.715667085 +0100 CET /bin
  drwxr-xr-x  1000  1000      0 2016-11-11 01:47:56.442413827 +0100 CET /home/pi/go/bin
  drwxr-xr-x  1000  1000      0 2016-10-19 19:39:27 +0200 CEST /home/pi/go/misc/nacl/testdata/bin
  drwxr-xr-x  1000  1000      0 2016-11-11 01:40:53.031719366 +0100 CET /home/pi/go/pkg/bootstrap/bin
  drwxr-xr-x  1000  1000      0 2015-09-04 06:55:47 +0200 CEST /home/pi/go-linux-arm-bootstrap/bin

$ ./restic -r rest:http://qnap********:8000 ls -l 0eb7a75d
snapshot of [/ /boot] at 2017-03-02 03:49:02.056313104 +0100 CET:
-rwxr-xr-x   501    80    345 2016-02-04 10:42:20 /._etc
-rwxr-xr-x   501    80    345 2016-02-02 17:01:30 /._var
drwxr-xr-x     0     0      0 2017-01-29 13:21:28 /bin
-rwxr-xr-x     0     0 863400 2017-01-15 00:14:18 /bin/bash
-rwxr-xr-x     0     0  30300 2014-08-06 13:25:38 /bin/bunzip2
-rwxr-xr-x     0     0  30300 2014-08-06 13:25:38 /bin/bzcat
Lrwxrwxrwx     0     0      0 2016-04-20 00:18:34 /bin/bzcmp -> bzdiff
```

## Actual behavior
```shell
$ ./restic -r rest:http://qnap.********:8000 find -s 0eb7a75d bin
found 55 matching entries in snapshot 0eb7a75d607a6dffd4f9342e2524326b23151d57875131a368ceb9f17d2b51c3
  drwxr-xr-x     0     0      0 2017-01-29 13:21:28.715667085 +0100 CET bin
  drwxr-xr-x  1000  1000      0 2016-11-11 01:47:56.442413827 +0100 CET home/pi/go/bin
  drwxr-xr-x  1000  1000      0 2016-10-19 19:39:27 +0200 CEST home/pi/go/misc/nacl/testdata/bin
  drwxr-xr-x  1000  1000      0 2016-11-11 01:40:53.031719366 +0100 CET home/pi/go/pkg/bootstrap/bin
  drwxr-xr-x  1000  1000      0 2015-09-04 06:55:47 +0200 CEST home/pi/go-linux-arm-bootstrap/bin

$ ./restic -r rest:http://qnap********:8000 ls -l 0eb7a75d
snapshot of [/ /boot] at 2017-03-02 03:49:02.056313104 +0100 CET:
-rwxr-xr-x   501    80    345 2016-02-04 10:42:20 ._etc
-rwxr-xr-x   501    80    345 2016-02-02 17:01:30 ._var
drwxr-xr-x     0     0      0 2017-01-29 13:21:28 bin
-rwxr-xr-x     0     0 863400 2017-01-15 00:14:18 bin/bash
-rwxr-xr-x     0     0  30300 2014-08-06 13:25:38 bin/bunzip2
-rwxr-xr-x     0     0  30300 2014-08-06 13:25:38 bin/bzcat
Lrwxrwxrwx     0     0      0 2016-04-20 00:18:34 bin/bzcmp -> bzdiff
```

## Steps to reproduce the behavior
Just run find or ls on a repo.

### PR DESCRIPTION
**Pull request by middelink**
Closes #839

### CODE REVIEW

File: src/cmds/restic/integration_test.go [hunk 1]
```diff
@@ -516,23 +516,23 @@ func TestBackupExclude(t *testing.T) {
 		testRunBackup(t, []string{datadir}, opts, gopts)
 		snapshots, snapshotID := lastSnapshot(snapshots, loadSnapshotMap(t, gopts))
 		files := testRunLs(t, gopts, snapshotID)
-		Assert(t, includes(files, filepath.Join("testdata", "foo.tar.gz")),
+		Assert(t, includes(files, filepath.Join("/testdata", "foo.tar.gz")),
```
Comment (fd0) [inline src/cmds/restic/integration_test.go]: Hm, shouldn't this be rather:
```go
Assert(t, includes(files, filepath.Join(string(filepath.Separator), "testdata", "foo.tar.gz")),
```

What do you think?

**Review by fd0 — CHANGES_REQUESTED** [review]

File: src/cmds/restic/integration_test.go (hunk 1 above)
Comment (fd0) [inline src/cmds/restic/integration_test.go]: Huh, that's odd. It seems that `filepath.Join()` on Windows automatically converts `/testdata` to `\testdata`. That's why the test does not fail on Windows. I'd still prefer having the filepath separator in, for clarity. Thanks!

Comment (middelink) [inline src/cmds/restic/integration_test.go]: Might be more conforming, yes. Though weirdly, the one OS I expected to have issues here, succeeded with the test. I'll make the change.

Comment (fd0) [inline src/cmds/restic/integration_test.go]: Cool, thanks for taking the time!

**Review by fd0 — APPROVED** [review]

**Comment by fd0** [comment]
Thanks!

**Pull request merged**